From 97058f1311421c640c2617ae4b97b7a9208d05c8 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 11 Jan 2021 18:15:57 +0200
Subject: [PATCH] [SW-20719] pll: move div factor define to header

The HL_PLL_DIV_FACTOR_CMD define will be used directly inside the HBM
code while it needs to toggle (reduce) the configuration PLL as a W/A.
Prepare for HBM code addition and move the define out to header.

Change-Id: I42bee7175591393c4fbabdda1dc03d20c2ce06b1
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.c b/common/pll.c
index 7c80b49..591dd2b 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -18,7 +18,6 @@
 #define HL_PLL_LOCK_INTR	0x128
 #define HL_PLL_DATA_CHNG	0x130
 #define HL_PLL_RST		0x134
-#define HL_PLL_DIV_FACTOR_CMD	0x220
 #define HL_PLL_CLK_GATER	0x300
 #define HL_PLL_CLK_RLX		0x310
 
diff --git a/common/pll.h b/common/pll.h
index 31e494e..371bf25 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -9,6 +9,7 @@
 #define _HL_PLL_H_
 
 #define HL_PLL_DIV_FACTOR	0x200
+#define HL_PLL_DIV_FACTOR_CMD	0x220
 #define HL_PLL_DIV_SEL		0x280
 #define HL_PLL_DIV_EN		0x2a0
 #define SYS_REF_CLOCK		50000000ULL
